pkg/logger/logger_impl.go

```diff
@@ -186,6 +191,11 @@ func (l logrusEntryWrapper) SetReportCaller(flag bool) {
 	l.Logger.SetReportCaller(flag)
 }
 
+// TODO: remove this function when the Connect migration process to the new logging standard is complete.
```

Comment:
Here, and above on line 147, consider removing the Connect reference while keeping the comment.